app/views/prompts/edit.html.erb

```diff
@@ -9,6 +9,15 @@
 <!--/subnav-->
 
 <!--main content-->
+
+<% unless @prompt.errors.empty? %>
+    <div class="error">
+      <h4 class="heading"><%= ts("There were some problems with this submission. Please correct the mistakes below.") %></h4>
+    </div>
+  <% end %>
+
+<%= error_messages_for @prompt %>
```

Comment:
Can you add a test for this? You can create a valid prompt meme sign-up similar to:

https://github.com/otwcode/otwarchive/blob/030fa4607d3c59fba2cacbe4e9c99b3c49fbc1cb/features/prompt_memes_a/challenge_promptmeme_setup.feature#L111-L129

Then have the test edit the prompt, delete some info, and check that the errors show up.

Reply:
Thanks for all your feedback! One question: the challenge_promptmeme_setup feature seems to test for a variety of user actions already, among them "Edit individual prompt via signup show". Is it okay to add an "Update individual prompt via signup" scenario here, or should it be in a separate feature?

Reply:
Nice find! I think adding to the existing scenario "Edit individual prompt via signup show" sounds good.

Reply:
Done, and I ran it successfully on local.

Reply:
Code Climate complained about similarities between 2 controllers that can both edit sign-ups. That's not something we need to deal with here, so I've set it to passing.